src/mlpack/methods/ann/layer/max_pooling_impl.hpp

```diff
@@ -31,24 +31,82 @@ MaxPooling<InputDataType, OutputDataType>::MaxPooling(
     const size_t kernelHeight,
     const size_t strideWidth,
     const size_t strideHeight,
-    const bool floor) :
+    const bool floor,
+    const size_t inputWidth,
+    const size_t inputHeight,
+    const size_t padW,
+    const size_t padH,
+    const std::string paddingType) :
+    MaxPooling(
+    kernelWidth,
```

Comment:
Hi @tejasvi, Could you also indent constructor call. You can refer to this [example](https://github.com/mlpack/mlpack/blob/b8db9ba0d97f361f211ec82607cf8dfd9d8361f2/src/mlpack/methods/ann/layer/convolution_impl.hpp#L65_).
Thanks.

Reply:
Done.